README.md

```diff
@@ -23,10 +25,61 @@ ember install ember-frost-object-details
 ```
 
 ## API
-Coming soon
+|Component| Attribute | Type | Value | Description |
+|---------| --------- | ---- | ----- | ----------- |
+| `{{frost-object-details}}` | `defaultRoute` | `string` | | Default entry route for object details view (Mandatory) |
+| `{{routes.view}}` | `detailsRouteIndex` | `number` | |  Provide left/right slide animation when transition between two view routes based on comparing their detailsRouteIndex value (Optional) |
+| `{{routes.related}}` | `icon` | `string` | | The name of the icon to display(optional) |
 
 ## Examples
-Coming soon
+### frost-object-details
```

Comment:
Could add something about what a 'view' route vs 'related' route means? I am just reading this for the first time and I'm not sure what part of the object details page they are referring to.